packages/mui-material/src/Tooltip/Tooltip.js

```diff
@@ -263,6 +263,8 @@ const Tooltip = React.forwardRef(function Tooltip(inProps, ref) {
     ...other
   } = props;
 
+  const children = React.isValidElement(childrenProp) ? childrenProp : <span>{childrenProp}</span>;
```

Comment:
```suggestion
  // to prevent runtime errors, developers will need to provide a child as a React element anyway.
  const children = React.isValidElement(childrenProp) ? childrenProp : <span>{childrenProp}</span>;
```

Is my understanding correct?